gql/schema.graphql

```diff
@@ -9189,7 +9189,7 @@ enum ServiceHandlerType {
   HTTP
 
   """
-  Handle TLS for PostgreSQL connections
+  Handle TLS for Postgres connections
```

Comment:
I think this file is generated from `web`, so you may need to change this text over there too to prevent it from getting overwritten in the future.